### DISCUSSION TITLE
Icons for Admin Center

### DISCUSSION THREAD

**Discussion by johannaweintraub** [twilio-labs/paste#732] [Request: Icon] [answered]
Admin Center will be changing their nav to use the new Components and will be adding icons. Attached are the icons they will use and the final design. Can these be added to Paste?

SVG Icons from Streamline can be found here:
https://www.dropbox.com/sh/glywbp60ntcwyug/AACuE1oIs2eBFXesL5zmHFkfa?dl=0

Final Design: https://share.goabstract.com/2cbd141f-e27e-4d18-80cf-0369cc0eecf3?mode=design&sha=1aa507ad56bf38fcf0cd4683fb6188f746d7eaef

**Comment by richbachman** [comment]
Hi @johannaweintraub. Thanks for sending this over. We'll get them added. Do you have a specific timeline on when you need them added?

**Reply by johannaweintraub** [reply]
Probably by end of October would be safe for us. Thank you!

**Reply by richbachman** [reply]
Thanks. I'll get a ticket added to our backlog.

**Comment by corinne-nagel** [comment, marked as answer]
Hi @johannaweintraub these icons are now added and can be viewed here:
https://paste.twilio.design/icons
As a reminder, if you need something on a defined timeline, we would love for you to contribute using the guidelines here: https://paste.twilio.design/icons/how-to-add-an-icon/ Let us know if you have any questions!

**Comment by johannaweintraub** [comment]
Awesome! Thanks very much.
